tiatoolbox/cli.py

```diff
@@ -119,22 +126,33 @@ def slide_info(wsi_input, output_dir, file_types, mode, workers=None):
 )
 def read_region(wsi_input, region, level, output_path, mode):
     """Reads a region in an whole slide image as specified"""
-    if all(region):
+    if not region:
```

Comment:
Is this checking if that region is not None? I would consider changing it to `if region is not None:` if so as it is more clear.

Reply:
Its to check empty tuple "()".